layouts/ucf-news-card.php

```diff
@@ -36,26 +36,26 @@ function ucf_news_display_card_title( $item, $title, $display_type ) {
 }
 
 if ( ! function_exists( 'ucf_news_display_card' ) ) {
-	function ucf_news_display_card( $items, $title, $perrow, $display_type ) {
+	function ucf_news_display_card( $items, $title, $per_row, $display_type ) {
 		if ( ! is_array( $items ) ) { $items = array( $items ); }
 		ob_start();
 
-		echo '<div class="card-deck">';
+		echo '<div class="ucf-news-card-deck card-deck">';
```

Comment:
I left both classes in so that the plugin would work without the included css when athena was available.